### ISSUE TITLE
Fan mode

### ISSUE THREAD

**Issue by rbflurry** [nkgilley/python-ecobee-api#26]
I am trying to implement the Fan mode into Homeassistant but it does not appear to work.
I can see the following body get passed to the ecobee api and I get the 200 response but it actually doesn't work. Any ideas?

{"selection":{"selectionType":"thermostats","selectionMatch":"Removed"},"thermostat":{"settings":{"vent":"on"}}}

<Response [200]>

{
  "status": {
    "code": 0,
    "message": ""
  }
}

**Comment by rbflurry** [comment]
Nevermind it does not seem to be supported. 

https://getsatisfaction.com/api/topics/not-able-to-control-the-ventilator-from-the-apis

**Comment by uchagani** [comment]
It looks like you can turn the fan "on" and "auto" via the ```setHold``` function and passing in a "fan" option.  The request looks like this:

```json
{
  "selection": {
    "selectionType":"thermostats",
    "selectionMatch":"redacted"
  },
  "functions": [
    {
      "type":"setHold",
      "params":{
        "holdType":"nextTransition",
        "heatHoldTemp":650,
        "coolHoldTemp":700,
        "fan": "on"
      }
    }
  ]
} 
```

@nkgilley would you be interested in adding this to the ```set_hold_temp``` method?  This way we can turn the fan on whenever we want.  If so, I can do the work and push up a PR.

**Comment by nkgilley** [comment]
Sure thing, PR welcome :)

**Comment by uchagani** [comment]
implemented in #29
